### ISSUE TITLE
calling sitk function with int argument does not work with Python 3

### ISSUE THREAD

**Issue by haarburger** [Radiomics/pyradiomics#204]
@jcfr When using Python 3.5, resampling as implemented in [imageoperations.py#L187](https://github.com/Radiomics/pyradiomics/blob/cc788414606f23b1bbd5c6327035ffb9c7bcadc3/radiomics/imageoperations.py#L187) does not work.

[sitk.ResampleImageFilter](https://itk.org/SimpleITKDoxygen/html/classitk_1_1simple_1_1ResampleImageFilter.html#a54442931f33522ca0f7b9a234dc0dba6) accepts uint32_t arguments. In Python 2.7 this works fine with passing `np.int`. However, in Python 3 this is not working anymore. 

Example:
```
rif = sitk.ResampleImageFilter()
rif.SetSize(np.array([1, 2, 3], dtype='int'))

TypeError: in method 'ResampleImageFilter_SetSize', argument 2 of type 'std::vector< uint32_t,std::allocator< uint32_t > > const &'
```
Which numpy type whould result in uint32_t for both python 2 and 3? With `np.uint32` I get the same error.

Converting the array to python `int` with `arr = [int(x) for x in arr]` resolves the issue but is ugly...

**Comment by JoostJM** [comment]
@haarburger, thanks for the heads up! This conversion was giving us trouble before (and fixed it by casting to 'int', but apparently doesn't work for python 3). We'll look into it to make it python 3 compatible. If you find a solution in the meantime, we'd welcome your contribution.

**Comment by haarburger** [comment]
`np.array([1, 2, 3], dtype='int').tolist()` does the trick. I'll create a PR shortly.

**Comment by jcfr** [comment]
> rif = sitk.ResampleImageFilter()
> rif.SetSize(np.array([1, 2, 3], dtype='int'))
> TypeError: in method 'ResampleImageFilter_SetSize', argument 2 of type 'std::vector< uint32_t,std::allocator< uint32_t > > const &'

> np.array([1, 2, 3], dtype='int').tolist() does the trick. I'll create a PR shortly.

@blowekamp Would it make sense to improve the wrapping layer to directly understand numpy array as parameter ?

**Comment by blowekamp** [comment]
The swig wrapping layer does under stand numpy array.  It does not implicitly convert numpy signed integer types to unsigned array types.

**Comment by haarburger** [comment]
@blowekamp Actually, `np.array([1, 2, 3], dtype='uint32')` was the first thing I tried, but it doesn't work.

**Comment by JoostJM** [comment]
@haarburger, did you also try 'uint'?

**Comment by haarburger** [comment]
Yes, doesn't work either.

**Comment by blowekamp** [comment]
I will have to look into the further. Can you please create a [SimpleITK Issue](https://github.com/SimpleITK/SimpleITK/issues), and include the SimpleITK version, OS, Python Distribute, etc.. where the problem has occurred?

Have you checked if SimpleITK 1.0rc1 has the same issue?

**Comment by haarburger** [comment]
@blowekamp the problem also occurs with 1.0rc1. I created a SimpleITK issue.

**Comment by JoostJM** [comment]
Addressed in #205
